web/src/components/common/DynamicTable.tsx

```diff
@@ -0,0 +1,43 @@
+import { EdsProvider, Table } from '@equinor/eds-core-react'
+
+export type TDynamicTableInput = {
+  // Keys will be used as headers. The array for each key will be the data in the column
+  [key: string]: Array<string>
+}
+export const DynamicTable = (props: {
+  input: TDynamicTableInput
+}): JSX.Element => {
+  const { input } = props
+
+  function createTableBody() {
```

Comment:
The function doesn't really create a table body tho.. maybe we should find a more suitable name for it

Reply:
Agreed. Renamed

Reply:
I think you forgot?

Reply:
Damn rebase....